lib/gollum/public/gollum/javascript/editor/langs/asciidoc.js

```diff
@@ -113,7 +113,34 @@ var AsciiDoc = {
                                   }
                                 });
                               }
-                            }
+                            },
+                            
+  'function-critic-accept' : {
```

Comment:
Having to repeat these functions is obviously not very DRY (by the looks of it, there are no differences for the various language definitions, right?). Do you see a way of making it more so? If not, then never mind -- this PR shouldn't be held up because of the way language definitions are currently working.